### PR TITLE
Test branch 2

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by shajanjp** [comment]
@kurticusj 
Your pin should be in this format.
```
---
githubHandle: <YOUR_USER_NAME>
latitude: <LATITUDE>
longitude: <LONGITUDE>
---
```

File: .jso [hunk 1]
```diff
@@ -0,0 +1,5 @@
+----
+githubHandle: Kurticusj.jso
+Longitude = 37.4138° N
+Latitude = 79.1422° W
+----
```
Comment (hectorsector) [inline .jso]: @kurticusj remove this file altogether

File: _pins/Kurticusj.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+----
+githubHandle:Kurticusj
```
Comment (hectorsector) [inline _pins/Kurticusj.json]: @kurticusj add a space between the `:` and your username

**Comment by hollenberry** [comment]
Closing this out as this user has updated their profile to reflect `Deleted User`. 

Please feel free to re-open if this is in error.

**Pull request closed without merging**